yt/data_objects/static_output.py

```diff
@@ -1971,6 +1971,17 @@ def __init__(
             default_species_fields=default_species_fields,
         )
 
+    def _disable_periodicity_for_bbox(self):
+        # This is called when a bounding box is explicitly set by the
+        # user--currently in these circumstances periodicity behaves
```

Comment:
```suggestion
        # user--previously in these circumstances periodicity behaved
```

Reply:
It's still true that periodicity behaves oddly if we apply a bounding box--I don't think the change in tense is what we want here.

Reply:
ah I see, nevermind